src/endpoints/PricingTables/types.ts

```diff
@@ -36,3 +37,22 @@ export interface AddOn {
     display_name: string;
     prices: Price[];
 }
+
+export interface Row {
+    display_name: string;
+    description: string | null;
+    link: string | null;
+    area: string | null;
+    status: 'beta' | null;
+    is_key_item: boolean;
+    cells: {
+        starter?: Cell;
+        core?: Cell;
+        premium?: Cell;
```

Comment:
@marko-ogg you could probably do something like 
```typescript
export interface Row {
...
cells: Record<string, Cell>
}
```

Reply:
I also would create a type for the plans
```
type PlanNames = 'starter' | 'core' | 'premium'
...
cells: Record<PlanNames, Cell>;
```

Reply:
Thanks guys for suggestions!

@aspirisen why do you think plan types are necessary? Those are just pricing table option IDs. Seems to me like it will not be necessary for using (rendering) the table, since you will be accessing these properties based on contained pricing option IDs in `options` property.

Just want to understand how specifying these plan names helps?

Reply:
hm, yeah, good point if we are not going to treat some plans in a special way

Reply:
Yeah. We shouldn't, everything should be controlled by backend and provided in the API response. Frontend should only be rendering (representing) the data.

I try to avoid as much as possible any plan name checks on frontend side, since that is what we had before I dropped the plan level concept earlier.